project_stock/models/project_project.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = "project.project"
+
+    picking_type_id = fields.Many2one(
+        comodel_name="stock.picking.type",
+        string="Operation Type",
+        readonly=False,
+        domain="[('company_id', '=', company_id)]",
```

Comment:
According to https://www.odoo.com/documentation/13.0/developer/howtos/company.html#multi-company-consistency it's not necessary.

Reply:
Please restore it. This is needed as you can be logged in (allowed companies) in several companies at the same time, and this makes that you select the proper records according document's company. In fact, this is needed only since 13.0 new multi-company system. Check several examples in Odoo core:

https://github.com/odoo/odoo/blob/9d518f847d2204fc5159a92bc4e4de655acf88fa/addons/account/models/account_move.py#L121
https://github.com/odoo/odoo/blob/9d518f847d2204fc5159a92bc4e4de655acf88fa/addons/sale/models/sale.py#L205

@cesar-tecnativa please be sure before asking changes